setup.cfg

```diff
@@ -49,6 +49,7 @@ keywords =
 
 [options]
 install_requires =
+    sqlalchemy<2.0.0
```

Comment:
Can we combine merging this with a new issue to unpin, that contains the link to the issue that is is the reason why pinning is important?

Reply:
We could but that means Charlie will have to wait .... your call.

Reply:
I meant, we can merge your proposal now, but create a second issue to unpin so we don't forget!

Reply:
Aah! gotcha! [Done!](https://github.com/mapping-commons/sssom-py/issues/343)